catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -256,24 +285,36 @@ protected Object executeWithSubject() throws Exception {
     }
 
     console.println("Export complete.");
-    console.println("Exported to: " + zipFile.getFile().getCanonicalPath());
+    console.println("Exported to: " + outputFile.getCanonicalPath());
     return null;
   }
 
-  private File initOutputFile(String output) {
-    String resolvedOutput;
-    File initialOutputFile = new File(output);
-    if (initialOutputFile.isDirectory()) {
-      // If directory was specified, auto generate file name
-      resolvedOutput = Paths.get(initialOutputFile.getPath(), FILE_NAMER.get()).toString();
-    } else {
-      resolvedOutput = output;
-    }
+  private void auditRecords(List<ExportItem> exportedItems) {
+    AtomicInteger counter = new AtomicInteger();
+    exportedItems
+        .stream()
+        .map(ExportItem::getId)
+        .distinct()
+        .collect(Collectors.groupingBy(e -> logPartition(e, counter)))
+        .values()
+        .forEach(this::writePartitionToLog);
+  }
 
-    return new File(resolvedOutput);
+  private int logPartition(String e, AtomicInteger counter) {
+    //  Number of bytes that can be sent is 65,507 so give a 1000 buffer to account for any preface
+    final int log4jMaxBufSize = 64507;
```

Comment:
✏️ personal opinion, I would **vastly** underestimate the size. like 10,000 instead of getting anywhere close to the 65k limit. Even 1000 away but still possibly having extra things in there scares me. :P 

In fact now I think about it, Theoretically we should be doing this for any audit log being sent that could grow large -- maybe we should (in the future or another ticket) look into protecting the audit logger by automatically partitioning too large of singular audits.

Reply:
I feel like 10000 might be excessive especially when considering that the delimiters are taken into account there's only 2 characters for [ ] and then 9998 for randomness and the security stuff that preludes it. Perhaps 3000 would be a nice enough buffer?

Reply:
I meant only 10,000 characters **allowed**, effectively a 64,507-10,000 length buffer 😂

Reply:
Because honestly, a 10,000 single length line is going to be so atrociously unreadable anyways ... if these are even supposed to be human readable... 🤷‍♂️

Reply:
You can leave it as is as long as you're confident it'll be 👌  Just the buffer in the first place scared me to think we might be dealing with some amount of variability 😄

Reply:
I'm fairly confident that there won't be any errors with it. The buffer is supposed to account for everything that comes before the id's in the log record 
IE: ```[INFO ] 2018-09-12T12:39:49,213 | n --delete=false | securityLogger  |  Subject: CN=localhost Ids of exported metacards and content:_``` since that can be variable size.